### PR TITLE
Add some WebGPU demos

### PR DESCRIPTION
**Pull request by gyagp**
Add some WebGPU demos from Transformers.js and TVM. 
Adjust the samples for better maintenance.

### CODE REVIEW

**Comment by gyagp** [comment]
@NingW101 @ibelem, PTAL

File: config.js [hunk 1]
```diff
@@ -20,7 +20,7 @@
  *                                                             *
  ***************************************************************/
 
-export const USE_REMOTE_MODELS = false;
+export const USE_REMOTE_MODELS = true;
```
Comment (gyagp) [inline config.js:23]: It seem this line is auto changed. By "npm run prod:use-remote-models"?

File: js/main.js [hunk 2]
```diff
@@ -13,236 +13,312 @@ const baseUrl =
...
 
-const SUPPORTED_DEVICES = {
-  GPU: "GPU",
+const DEVICES = {
   CPU: "CPU",
+  GPU: "GPU",
   NPU: "NPU"
 };
 
+// To add a new sample, please put <id>.png and <id>.mp4 in the public/assets folder.
```
Comment (gyagp) [inline js/main.js:27]: I hope all the fields are mandatory in the future, to make the maintenance easier.

File: js/main.js [hunk 3]
```diff
@@ -282,56 +360,29 @@ function constructSampleHTML(samples) {
...
-                    sample.supportHdw.length > 0
-                      ? ` <div
-                  class="flex rounded-md font-semibold bg-sky-600/80 px-2 py-1 text-stone-50 ring-1 ring-inset text-sm ring-stone-500/10 w-auto"
-                >
-                  ${sample.supportHdw.join(" · ")}
-                </div>`
-                      : ``
-                  }
-
+              <div class="flex rounded-md font-semibold bg-sky-600/80 px-2 py-1 text-stone-50 ring-1 ring-inset text-sm ring-stone-500/10 w-auto">
```
Comment (gyagp) [inline js/main.js:374]: Adjust the UI a bit, as info like model, webApi, framework and HW cover a lot of pictures and videos.

File: samples/image_to_text/image_to_text.js [hunk 4]
```diff
@@ -16,7 +16,7 @@ import {
   getElementId4Resource,
   removeHiddenClass
 } from "../common/utility.js";
-import exampleImgPath from "/assets/cats.jpg";
+import exampleImgPath from "/assets/cats.png";
```
Comment (gyagp) [inline samples/image_to_text/image_to_text.js:19]: Hope to unify all the picture format to .png

File: js/main.js [hunk 5]
```diff
@@ -13,236 +13,312 @@ const baseUrl =
...
 let SAMPLES = [
+  // WASM
   {
-    title: "Phi-3",
-    desc: "A private and powerful AI chatbot that runs locally in your browser.",
-    poster: `${baseUrl}assets/phi3.png`,
-    sampleUrl: "./samples/phi3-webgpu/index.html",
-    model: "Phi-3-mini-4k-instruct",
-    webApi: [BACKENDS.WebGPU],
+    id: "wasm_image_to_text",
```
Comment (gyagp) [inline js/main.js:31]: id is added for unified names of picture and video. In addition, id starts with backend name now for better maintenance.

File: config.js (hunk 1 above)
Comment (NingW101) [reply config.js:23]: Yes, we'd better not track the modification on the commit

Comment (gyagp) [reply config.js]: Reverted the change in a new PR. PTAL.

**Review by NingW101 — APPROVED** [review]
LTGM, Thanks so much~

**Pull request merged**